config/config.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+import (
+	"github.com/joho/godotenv"
+	"log"
+)
+
+func init() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal(err)
+		log.Fatal("Error loading .env file")
```

Comment:
This error message is redundant as `err` already contains enough information. 

Moreover, this line will never be reached as the previous `log.Fatal(err)` calls `os.Exit(1)`. A possible solution could be `log.Fatal("Error load .env file ", err)`.

Reply:
Agreed